libr/core/cmd_anal.c

```diff
@@ -5065,15 +5118,17 @@ static void cmd_anal_aav(RCore *core, const char *input) {
 	char *arg = strchr (input, ' ');
 	if (arg) {
 		ptr = r_num_math (core->num, arg + 1);
-		s = r_io_section_vget (core->io, ptr);
+		secs = r_io_section_vget_secs_at (core->io, ptr);
+		s = secs ? ls_pop (secs) : NULL;
```

Comment:
I have a commit in loca to bring back r_io_section_get_vaddr to avoid such pattern. I didn't commit yet